css/layout-css/small-card-style.upload.css

```diff
@@ -985,7 +982,6 @@
 .small-card-detail-overlay .small-card-list-detail-button a:active,
 .small-card-detail-overlay .small-card-list-detail-button a:active:hover,
 .small-card-detail-overlay .small-card-list-detail-button a:active:focus {
-  color: #333333;
```

Comment:
Are we sure that removing this will not make the colour of the link some other colour when hovered, in focus, and/or active?

Reply:
@hcarneiro It doesn't change color on hover or active on link.
https://share.getcloudapp.com/7Ku0nzkm